ir/src/transition_constraints/graph.rs

```diff
@@ -231,7 +299,7 @@ impl Node {
 /// A transition constraint operation or value reference.
 #[derive(Debug, Eq, PartialEq)]
 pub enum Operation {
-    Const(u64),
+    Constant(ConstantValue),
```

Comment:
I would add doc comment for `Constant` in the same way we do for other variants here.